core/src/main/java/org/elasticsearch/discovery/zen/ping/unicast/UnicastZenPing.java

```diff
@@ -64,7 +64,7 @@
 
     public static final String ACTION_NAME = "internal:discovery/zen/unicast";
 
-    public static final int LIMIT_PORTS_COUNT = 1;
+    public static final int LIMIT_PORTS_COUNT = 10;
```

Comment:
Cool! If we merge it I think we can close https://github.com/elastic/elasticsearch/pull/8833 ! It will also help in discovery plugins.